src/logger.ts

```diff
@@ -14,33 +14,41 @@ const streams = {
 	info: fs.createWriteStream(`${root}/logs/info.log`)
 } as const;
 
+function getCurrentTimestamp(): string {
+	const date = new Date();
+	const year = date.getFullYear();
+	const month = String(date.getMonth() + 1).padStart(2, '0'); // Months are 0-indexed
```

Comment:
Minor nit. We use the comment syntax found in Better Comments ([marketplace.visualstudio.com/items?itemName=EdwinHuiSH.better-comments-next](https://marketplace.visualstudio.com/items?itemName=EdwinHuiSH.better-comments-next))

```suggestion
	const month = String(date.getMonth() + 1).padStart(2, '0'); // * Months are 0-indexed
```